File: domain/src/main/java/com/aceplus/domain/model/forApi/other/Location.java
package com.aceplus.domain.model.forApi.other;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by haker on 2/15/17.
 */
public class Location {
    @SerializedName("LocationId")
    @Expose
    private String locationId;
    @SerializedName("LocationNo")
    @Expose
    private String locationNo;
    @SerializedName("LocationName")
    @Expose
    private String locationName;
    @SerializedName("BranchId")
    @Expose
    private String branchId;
    @SerializedName("BranchNo")
    @Expose
    private String branchNo;

    public String getLocationId() {
        return locationId;
    }

    public void setLocationId(String locationId) {
        this.locationId = locationId;
    }

    public String getLocationNo() {
        return locationNo;
    }

    public void setLocationNo(String locationNo) {
        this.locationNo = locationNo;
    }

    public String getLocationName() {
        return locationName;
    }

    public void setLocationName(String locationName) {
        this.locationName = locationName;
    }

    public String getBranchId() {
        return branchId;
    }

    public void setBranchId(String branchId) {
        this.branchId = branchId;
    }

    public String getBranchNo() {
        return branchNo;
    }

    public void setBranchNo(String branchNo) {
        this.branchNo = branchNo;
    }
}

File: domain/src/main/java/com/aceplus/domain/entity/Location.kt
package com.aceplus.domain.entity

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


@Entity(tableName = "location")
class Location {

    @PrimaryKey
    @ColumnInfo(name = "location_id")
    @SerializedName("location_id")
    @Expose
    var location_id:  String = ""

    @ColumnInfo(name = "location_no")
    @SerializedName("location_no")
    @Expose
    var location_no:  String? = null

    @ColumnInfo(name = "location_name")
    @SerializedName("location_name")
    @Expose
    var location_name: String? = null

    @ColumnInfo(name = "branch_id")
    @SerializedName("branch_id")
    @Expose
    var branch_id:  String? = null

    @ColumnInfo(name = "branch_name")
    @SerializedName("branch_name")
    @Expose
    var branch_name: String? = null

}